sqlitecluster/SQLiteCommand.cpp

```diff
@@ -24,6 +24,11 @@ SQLiteCommand::SQLiteCommand(SData&& _request) :
                 break;
         }
     }
+
+    // If the request doesn't specify an execution time, default to right now.
+    if (!request.isSet("commandExecuteTime")) {
+        request["commandExecuteTime"] = to_string(STimeNow());
```

Comment:
it'd be great to figure out the automatic string casting from integers for `STable` -- I wasn't able to ever get that working.